src/mmw/apps/modeling/views.py

```diff
@@ -235,8 +237,17 @@ def choose_worker():
     def predicate(worker_name):
         return settings.STACK_COLOR in worker_name or 'debug' in worker_name
 
+    @retry(Exception, delay=0.5, backoff=2, tries=3)
```

Comment:
I'm fine with tweaking the parameters, but I think we need to determine at what threshold the call becomes less likely to succeed at. Does waiting 10 seconds give us a greater chance of success?

I don't think there is a clear answer, but given what I've seen of this particular interaction, any more than 2-3 tries it probably too much. Somewhere between 3-5 seconds may be the max as far as overall duration goes. If people with more insight feel differently, I'm open to hearing why.

/cc @jamesmcclain because he's the original author, and might have a better sense.